conda/core/package_cache.py

```diff
@@ -96,6 +97,18 @@ def is_extracted(self):
         return isdir(epd) and isfile(join(epd, 'info', 'index.json'))
 
     @property
+    @memoizemethod
+    def repodata_record(self):
```

Comment:
Fine with me, but there are several other properties in this class which also perform disk I/O, such as `md5sum`, `is_fetched`, `is_extracted`. I'd be happy to drop the `@property` from these as well; not quite sure how much adaptions this would require in the rest of the code.